CodeReview/src/nodes/ReviewComment.h

```diff
@@ -46,14 +46,23 @@ class CODEREVIEW_API ReviewComment : public Super<Model::CompositeNode>
 {
 	COMPOSITENODE_DECLARE_STANDARD_METHODS(ReviewComment)
 
-	ATTRIBUTE(Model::Text, date, setDate)
 	ATTRIBUTE(Comments::CommentNode, commentNode, setComment)
 
 	public:
-		ReviewComment(Comments::CommentNode* commentNode, Model::Text* date, Model::Node* parent=nullptr);
+		ReviewComment(Comments::CommentNode* commentNode, qint64 date, Model::Node* parent=nullptr);
+		qint64 date();
```

Comment:
Wait, that's not right, both the date and the username should be attributes. You want those to be persisted.